tests/generic_config_updater/test_syslog.py

```diff
@@ -0,0 +1,267 @@
+import logging
+import pytest
+
+from tests.common.helpers.assertions import pytest_assert
+from tests.common.config_reload import config_reload
+from tests.generic_config_updater.gu_utils import apply_patch, expect_op_success_and_reset_check, expect_res_success, expect_op_failure
+from tests.generic_config_updater.gu_utils import generate_tmpfile, delete_tmpfile
+
+pytestmark = [
+    pytest.mark.topology('t0'),
```

Comment:
>t0 [](http://example.com/codeflow?start=26&length=2)

syslog is a feature for any topology, not just `t0`.

Reply:
change to 'any'